packages/contracts/test/StabilityPoolTest.js

```diff
@@ -742,239 +696,6 @@ contract('StabilityPool', async accounts => {
       assert.equal(B_LQTYBalance_After, '0')
     })
 
-    it("provideToSP(), new deposit after past full withdrawal: depositor does not receive any LQTY rewards", async () => {
```

Comment:
do we still want this test?

Reply:
this test probably should have been left but would get removed in the next PR